rqt_py_common/src/rqt_py_common/topic_dict.py

```diff
@@ -68,8 +68,8 @@ def _recursive_create_field_dict(self, slot_name, field):
             'children': {},
         }
 
-        if hasattr(field, '__slots__'):
-            for child_slot_name in field.__slots__:
+        if hasattr(field, '_fields_and_field_types'):
+            for child_slot_name in field.get_fields_and_field_types().keys():
```

Comment:
This isn't a slot name anymore; it is a field name.  So I think we should rename this field to `field_name` (same below where we use it).

Reply:
Makes sense, I'll take that into account with the other PR's that solve similar issues.

Reply:
Done.